test/test_app.c

```diff
@@ -0,0 +1,555 @@
+/* Standard includes. */
```

Comment:
Let us discuss once about the right placeholder for test-apps

Reply:
It is decided that the tests will be moved out to a different repo and we can update and add them back later.